src/test/scala/org/ergoplatform/nodeView/wallet/scanning/ExternalApplicationSpecification.scala

```diff
@@ -0,0 +1,23 @@
+package org.ergoplatform.nodeView.wallet.scanning
+
+import io.circe.Json
+import org.ergoplatform.utils.ErgoPropertyTest
+import org.ergoplatform.utils.generators.ErgoTransactionGenerators
+
+class ExternalApplicationSpecification extends ErgoPropertyTest with ErgoTransactionGenerators {
+  import ExternalApplicationJsonCodecs._
+  import ScanningPredicateTestVectors._
+
+  property("external app req json serialization roundtrip") {
+    val req = ExternalAppRequest("Example application", complexOr)
```

Comment:
let's test roundtrips using generators, as it is done everywhere else.

Reply:
fixed in 3a05b757a2128107b010ce0d8667e3abd83611a2